configure.ac

```diff
@@ -785,6 +785,13 @@ fi
 
 fi
 
+case $host in
+   x86_64-*-linux*)
+  AC_CHECK_LIB([IPSec_MB],[sha1_one_block_avx2],LIBS=-lIPSec_MB, AC_MSG_ERROR(IPSec_MB missing))
+  AC_CHECK_LIB([IPSec_MB],[sha256_one_block_avx2],LIBS=-lIPSec_MB, AC_MSG_ERROR(IPSec_MB missing))
+  AC_CHECK_LIB([IPSec_MB],[sha512_one_block_avx2],LIBS=-lIPSec_MB, AC_MSG_ERROR(IPSec_MB missing))
+esac
+
```

Comment:
Question: since this adds a mandatory dependency for 64-bit linux, and there are issues with glibc compatibility for 1.14, perhaps this is better done on 1.21?

Reply:
@patricklodder perhaps you're right, I mean I'd really like this to work for 14.5, but yeah glibc issues shouldn't hold it up.. @michilumin and @rnicoll, any thoughts?

Reply:
Alternative 1: you can change the autoconf routine to set an enable flag if it's found, and make the configure-time dependency optional. When built with depends, it will then detect and enable for x86_64 linux, but not for the others? 

Alternative 2: you can also do a build flag, forget about depends and the release for a bit, and we can get some experience by running this in the field off a v1.14.5 tag built for-purpose. I'd totally put that on a few nodes to gather data on the performance if you'd like.

In both cases I'd need this to be reviewed thoroughly because this introduces a completely new dependency. @michilumin How far along is the actual review? If this is likely to be fully reviewed soon, and the I can drop some things and help fixing the glibc issue. I'm not going to do that if there's no in-depth review though.

Reply:
I can add a build flag for initial support, and I'm definitely interested in the peformance data you can gather.